tracer/resources/args_parser.py

```diff
@@ -16,7 +16,7 @@
 # 02110-1301, USA.
 #
 
-import argparse
+import argparse, argcomplete
```

Comment:
I disagree, even if we catch the `ImportError` what would be the point? `argcomplete` is going to get installed with every install.
I guess we could catch the `ImportError` and remove the dependency on `argcomplete` but if we do that we might as well not bother merging this..